src/ex4/server/clients/pokemon_client.js

```diff
@@ -0,0 +1 @@
+// The Pokemon Client (using axios) goes here
```

Comment:
why didn't you implement pokemonClient in this folder? please don't keep empty files in your PRs

Reply:
I will replace the file location